pocs-capstone/frontend/src/components/AnimateChoice.js

```diff
@@ -139,7 +171,7 @@ const AnimateChoice = () => {
                         endAt = {4}
                         onClick={() => handleClick(1)}
                     />
-                </div>
+                </div> */}
                 {showTextBox.map((isShown, index) => isShown && (
```

Comment:
since we're using a carousel we don't have to use the showTextBox state! It can look like this instead: 

``` 
<form onSubmit={(event) => handleSubmit(event, selectedIndex)}>
                    <input className="input" type="text" placeholder="Name your pet!" value={enteredText} onChange={textChangeHandler} />
                    <button className="button" type="submit">Submit</button>
 </form>

```